### ISSUE TITLE
Fix vulnerabilities in package.json

### ISSUE THREAD

**Issue by cristiano-belloni** [zambezi/caballo-vivo#28]
```
found 72 vulnerabilities (64 low, 1 moderate, 7 high)
run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by cristiano-belloni** [comment]
Partially solved by #29, but we still have some devdeps vulnerability

**Comment by gabrielmontagne** [comment]
Fixed by #52

**Comment by gabrielmontagne** [comment]
: ) 

![2021-02-04-180236_475x286_scrot](https://user-images.githubusercontent.com/95913/106928244-44213a80-6713-11eb-8ed7-19150e42dca6.png)
